NagorniyAleksandr/SQLTask/src/test/java/university/dao/crud/CRUDQuerySubjectTest.java

```diff
@@ -0,0 +1,218 @@
+package university.dao.crud;
+
+import org.h2.tools.RunScript;
+import org.junit.*;
+import university.dao.QueryCreator;
+import university.dao.QueryCreatorImpl;
+import university.exceptions.AppDBException;
+import university.jdbc.DBConnector;
+import university.jdbc.DBConnectorImpl;
+import university.models.Subject;
+import university.models.SubjectCategory;
+
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.hamcrest.core.IsNot.not;
+import static org.junit.Assert.*;
+
+public class CRUDQuerySubjectTest {
```

Comment:
Several tests repeat beforeClass logic
Create the parent test which will init the schema and data
Parent class will be inherited by child tests